packages/starksheet-cairo/deployments/devnetDocker/declarations.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
not sure what those files in starksheet-cairo/depoyments/devnetDocker achieve but they are required for the deployment to complete. I copied those from starksheet-cairo/depoyments/devnet as it follows the same configuration

Reply:
these are the latest class hash declared on a given network (here devnet). They are automatically generated by the deploy scripts. Because neither the devnet, nor this new config are saved, they should better be gitignored actually